README.md

```diff
@@ -1,73 +1,22 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
-# Project Name
-
-> Description the project.
-
+# Hello-microverse
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
+html and css
 
 ## Getting Started
 
 **This is an example of how you may give instructions on setting up your project locally.**
 **Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
```

Comment:
**[REQUIRED CHANGES]**

- Please, kindly remove this text content as they're not descriptive in telling the user how to have your project running locally on their machine. Instead, kindly add some instructions telling the users how to have your repo. follow the steps below:

To get a local copy up and running follow these simple example steps.

1. In the `Getting started` header, you should add some instructions on how a user can have your repo locally on their machine. (how to clone the repo, how to move into the cloned repo), 
2. Prerequisite: e.g `vscode editor`, a `web browser(chrome, etc)`.